utils/logging_configuration.py

```diff
@@ -45,7 +45,7 @@ class CustomGunicornLogFormatter(JsonFormatter):
         "levelname": "level",
         "r": "request",
         "a": "useragent",
-        "f": "referer",  # the biggest concern in my mind: should this be correctly spelled historically (one r) or correctly spelled linguistically (two r)
+        "f": "referrer",
```

Comment:
qq: does this break backwards compatibility with anything?

Reply:
nope -- this is only for the querying in our logs, which has been flowing for less than a day.